packages/webapp-libs/webapp-api-client/src/hooks/usePaginationQuery/__tests__/usePaginationQuery.hook.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import { act } from '@testing-library/react-hooks';
+
+import {
+  fillPaginationItemListQuery,
+  paginationListTestQuery,
+  paginationTestItemFactory,
+} from '../../../tests/factories';
+import { renderHook } from '../../../tests/utils/rendering';
+import { usePaginatedQuery } from '../usePaginatedQuery.hook';
+
+describe('usePaginationQuery: Hook', () => {
+  it('should paginate and fetch data in both directions', async () => {
```

Comment:
Why it's in single test? Please divide this into 3 separate tests:
- fetching page
- fetching next page
- fetching previous page